### DISCUSSION TITLE
How can I disable Global Search

### DISCUSSION THREAD

**Discussion by jwhoami** [filamentphp/filament#3420] [Help] [answered]
Hi,
How do I disable the Global Search?
Thanks

**Comment by jwhoami** [comment, marked as answer]
Found it
Just add to all your resources...

`protected static bool $isGloballySearchable = false;`

Though It would be nice to disable this via filament config.

**Comment by martingrondein-nls** [comment]
"Just add to all your resources" is a bit vague. I tried the above, but the Global Search is still visible. Any other tips?

**Reply by danharrin** [reply]
The search is visible, but which results are being returned? Still the ones from the disabled global search resources?

**Reply by jwhoami** [reply]
@martingrondein-nls  It worked for me, so there must be something different that you are doing.
If there is even a single resource which does not have the 

`protected static bool $isGloballySearchable = false;`

property, then the global search will be visible. Well that is my understanding and the reasoning behind "Just add to all your resources"

**Reply by danharrin** [reply]
yes @jwhoami is right, even third party resources

**Reply by danharrin** [reply]
Yeah but the record title attribute is used for other things too
